### DISCUSSION TITLE
type error after using useFormContext hook

### DISCUSSION THREAD

**Discussion by benevolentsocialplanner** [react-hook-form/react-hook-form#11582] [General]
const {setValue} = useFormContext();
  everytime this line of code triggered i get an error 
      setValue(field, val, {shouldDirty: true});
 Uncaught ReferenceError: setValue is not defined
    at handleChange (resourceSelectField.tsx:68:5)
    
    what should be the solution?
    i have tried to give it a type in many ways but didn't work.

**Comment by benevolentsocialplanner** [comment]
i think it's because setValue is reserved by javascript. haven't found the solution yet

**Reply by sujinleeme** [reply]
Did you wrap your form with `<FormProvider>`? It would be great if you share the codesandbox link or more code.

**Reply by benevolentsocialplanner** [reply]
yeah i wrapped it. 
working on codesandbox.
